src/sentry/event_manager.py

```diff
@@ -566,6 +566,28 @@ def save(self, project_id, raw=False, assume_normalized=False):
         if transaction_name:
             transaction_name = force_text(transaction_name)
 
+        # Right now the event type is the signal to skip the group. This
+        # is going to change a lot.
+        if event.get_event_type() == 'transaction':
+            create_group = False
+        else:
+            create_group = True
```

Comment:
nit: This doesn't mean that the event /will/ create a group, just that it /can/ create a group. Maybe this should be called `can_create_group` or something? That's not a great name either, but I think something more descriptive would help anybody who isn't familiar with this method from getting tripped up while reading it.

Reply:
renamed to issueless_event and inverted the condition